MVP Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```diff
@@ -299,3 +346,38 @@ final class FeedUIIntegrationTests: XCTestCase {
 		return UIImage.make(withColor: .red).pngData()!
 	}
 }
+
+// MARK: - Test Helpers
+extension FeedUIIntegrationTests {
+	private func test_errorView(_ errorView: ErrorView?,
```

Comment:
This method preform assertions, so it should be called 'assert' instead of 'test_errorView':

```suggestion
	private func assert(_ errorView: ErrorView?,
```

Reply:
Updated